lib/src/dom/dom.dart

```diff
@@ -1295,7 +1294,7 @@ extension type Document._(JSObject _) implements Node, JSObject {
   /// method of the [Document] interface returns a
   /// [CaretPosition] object, containing the DOM node, along with the caret and
   /// caret's character offset within that node.
-  external CaretPosition? caretPositionFromPoint(
+  external JSObject? caretPositionFromPoint(
```

Comment:
that's annoying

Reply:
We *could* generate `CaretPosition`, but then its members will likely all be hidden, so I'm not sure if that's better or worse than just returning a `JSObject`. If `CaretPosition` also is renamed, then that's a breaking change (in this specific case, presumably `caretPositionFromPoint` would be renamed as well, so that's probably a non-issue). We could also generate its members, but that also comes with the cost of potentially more breaking changes in the future.